portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/additional-information.component.ts

```diff
@@ -383,6 +389,16 @@ export class AdditionalInformationComponent extends FilesStepComponent implement
     }
   }
 
+  onChangeArea(id: string, event: Event) {
+    const input = event.target as HTMLInputElement;
+    const structure = this.proposedStructures.find((structure) => structure.id === id);
+    if (!structure) {
+      console.error('Failed to find structure');
+      return;
+    }
+    return this.setStructureAreaInput(structure, input.value);
```

Comment:
`setStructureAreaInput` doesn't return a value, so we don't really need to return here.

Reply:
makes sense !